csvcubed/csvcubed/models/cube/qb/catalog.py

```diff
@@ -70,7 +72,7 @@ def configure_dcat_dataset(self, dataset: dcat.Dataset) -> None:
         dataset.license = self.license_uri
         dataset.creator = self.creator_uri
         dataset.publisher = self.publisher_uri
-        dataset.landing_page = self.landing_page_uri ########################################
+        dataset.landing_page = self.landing_page_uris ########################################
```

Comment:
I'd be tempted to add them individually instead of overwriting something that's already there.